internal/cmd/request.go

```diff
@@ -8,34 +8,31 @@ import (
 	"github.com/edulinq/autograder/internal/api/core"
 	"github.com/edulinq/autograder/internal/api/server"
 	"github.com/edulinq/autograder/internal/common"
+	"github.com/edulinq/autograder/internal/log"
 	"github.com/edulinq/autograder/internal/util"
 )
 
 type CommonOptions struct {
-	Verbose bool `help:"Use verbose output to show full request/response without specific formatting." default:"false"`
+	Verbose bool `help:"Use verbose output to show the full request/response. Output may not adhere to a valid format/filetype." default:"false"`
```

Comment:
The wording here is strange.
When writing help for an option, we normally don't reference the option itself:
```
grep -R '`help:' internal cmd
```

I still don't think that the last part here would be clear to users.